desktop/ui/src/main/java/org/datacleaner/widgets/visualization/JobGraph.java

```diff
@@ -278,9 +284,31 @@ public void paint(Graphics g) {
                             + "So add one or more such components.";
                     imagePath = "images/window/canvas-bg-plus.png";
                 } else {
-                    title = null;
+                    title=null;
                     subTitle = null;
                     imagePath = null;
+                    
+                    try {
+                        if (!_analysisJobBuilder.isConfigured(true)) {
+                            subTitle="Job is not correctly configured";
+                            warning=true; 
+                        }
+                    } catch (Exception ex) {
+                        logger.debug("Job not correctly configured", ex);
+                        final String errorMessage;
+                        if (ex instanceof UnconfiguredConfiguredPropertyException) {
+                            UnconfiguredConfiguredPropertyException unconfiguredConfiguredPropertyException = (UnconfiguredConfiguredPropertyException) ex;
+                            ConfiguredPropertyDescriptor configuredProperty = unconfiguredConfiguredPropertyException
+                                    .getConfiguredProperty();
+                            ComponentBuilder componentBuilder = unconfiguredConfiguredPropertyException.getComponentBuilder();
+                            errorMessage = "Property '" + configuredProperty.getName() + "' in "
+                                    + LabelUtils.getLabel(componentBuilder) + " is not set!";
```

Comment:
Too technical for this space. We want to make it very friendly in the canvas' big text. So.
For the main title use a standard (but friendly, not "robot language") message about some component not being in a good state.
Keep the details to the subtitle.